contracts/handler/DefaultCallbackHandler.sol

```diff
@@ -0,0 +1,40 @@
+pragma solidity ^0.5.0;
+
+import "../interfaces/ERC1155TokenReceiver.sol";
+import "../interfaces/ERC721TokenReceiver.sol";
+import "../interfaces/ERC777TokensRecipient.sol";
+
+/// @title Default Callback Handler - returns true for known token callbacks
+/// @author Richard Meissner - <stefan@gnosis.pm>
+contract DefaultCallbackHandler is ERC1155TokenReceiver, ERC777TokensRecipient, ERC721TokenReceiver {
```

Comment:
I would use this as a simple first version that can be used for all safes. I would move this class to the modules repo once I move all other modules too.